### ISSUE TITLE
Not working in Desktop version maui

### ISSUE THREAD

**Issue by IMAZHARQ** [theadin/TineMCEDemo#1]
I checked this project my side not working in desktop version

**Comment by theadin** [comment]
Yes, I'm afraid this is a limitation of the technologies being used (EJL.MauiHybridWebView) and not in how the project is created. The author EJL.MauiHybridWebView of should be asked about this
